packages/twenty-server/src/workspace/messaging/services/messaging-utils.service.ts

```diff
@@ -199,29 +201,34 @@ export class MessagingUtilsService {
   }
 
   public async deleteMessages(
-    messagesDeleted: string[],
```

Comment:
I am not sure this function is properly named. From what I understand, the messages are already deleted and it deletes all entities around those messages if needed?

Reply:
These are the messages deleted from gmail, and we need to delete them in the db

Reply:
The input takes external messages IDs and they come from Gmail api telling us that they have been deleted in their side, we use this list to delete messages on our side as well